recipes/opencv/all/conanfile.py

```diff
@@ -0,0 +1,214 @@
+            tools.replace_in_file(find_openexr, r'set(OPENEXR_ROOT "")', "")
+            tools.replace_in_file(find_openexr, "SET(OPENEXR_LIBSEARCH_SUFFIXES x64/Release x64 x64/Debug)", "")
+            tools.replace_in_file(find_openexr, "SET(OPENEXR_LIBSEARCH_SUFFIXES Win32/Release Win32 Win32/Debug)", "")
+
+            def openexr_library_names(name):
+                # OpenEXR library may have different names, depends on namespace versioning, static, debug, etc.
+                reference = str(self.requires["openexr"])
+                version_name = reference.split("@")[0]
+                version = version_name.split("/")[1]
+                version_tokens = version.split(".")
+                major, minor = version_tokens[0], version_tokens[1]
+                suffix = "%s_%s" % (major, minor)
+                names = ["%s-%s" % (name, suffix),
+                         "%s-%s_s" % (name, suffix),
+                         "%s-%s_d" % (name, suffix),
+                         "%s-%s_s_d" % (name, suffix),
+                         "%s" % name,
+                         "%s_s" % name,
+                         "%s_d" % name,
+                         "%s_s_d" % name]
+                return " ".join(names)
+
+            for lib in ["Half", "Iex", "Imath", "IlmImf", "IlmThread"]:
+                tools.replace_in_file(find_openexr, "NAMES %s" % lib, "NAMES %s" % openexr_library_names(lib))
+
+        for cascade in ["lbpcascades", "haarcascades"]:
+            tools.replace_in_file(os.path.join(self._source_subfolder, "data", "CMakeLists.txt"),
+                                  "share/OpenCV/%s" % cascade, "res/%s" % cascade)
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.definitions["BUILD_EXAMPLES"] = False
+        self._cmake.definitions["BUILD_DOCS"] = False
+        self._cmake.definitions["BUILD_TESTS"] = False
+        self._cmake.definitions["BUILD_PACKAGE"] = False
+        self._cmake.definitions["BUILD_PERF_TESTS"] = False
+        self._cmake.definitions["BUILD_opencv_apps"] = False
+        self._cmake.definitions["BUILD_opencv_java"] = False
+        self._cmake.definitions["BUILD_ZLIB"] = False
+        self._cmake.definitions["BUILD_JPEG"] = False
+        self._cmake.definitions["BUILD_PNG"] = False
+        self._cmake.definitions["BUILD_TIFF"] = False
+        self._cmake.definitions["BUILD_JASPER"] = False
+        self._cmake.definitions["BUILD_OPENEXR"] = False
+
+        self._cmake.definitions["WITH_CUFFT"] = False
+        self._cmake.definitions["WITH_CUBLAS"] = False
+        self._cmake.definitions["WITH_NVCUVID"] = False
+        self._cmake.definitions["WITH_FFMPEG"] = False
+        self._cmake.definitions["WITH_GSTREAMER"] = False
+        self._cmake.definitions["WITH_OPENCL"] = False
+        self._cmake.definitions["WITH_CUDA"] = False
+        self._cmake.definitions["WITH_ZLIB"] = True
```

Comment:
zlib is also available on cci

Reply:
Yes, and also listed here in this recipe: https://github.com/conan-io/conan-center-index/pull/2743/files#diff-582d91edd9135a7ff7dcc411811b144dR51

Reply:
I misread. Must have been tired. Thanks!

Reply:
No problem! Thank you for reviewing, it makes this recipe even better!